tommyunited-bangkok.md

```diff
@@ -0,0 +1,5 @@
+# Bangkok, Thailand
```

Comment:
I'm a big fan of 🍺 any recommendations on places to grab a good pint in Bangkok?